code/game/objects/items.dm

```diff
@@ -65,7 +65,7 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 	var/equip_delay_other = 20 //In deciseconds, how long an item takes to put on another person
 	var/strip_delay = 40 //In deciseconds, how long an item takes to remove from another person
 	var/breakouttime = 0
-	var/list/materials
+	var/list/materials = list() //materials in this object, and the amount

```

Comment:
does it realy need to start of as a list? that costs a lot during init time

Reply:
i.e can it be lazy (ideally it should)

Reply:
I specifically said to not make an empty list T.T

Reply:
Why, that's going to cost a tonne

Reply:
No I'm agreeing with you here

Reply:
ohhhhhhhhhhh

Reply:
my bad, @ninjanomnom you told me not to make it an empty list on atom, I made the mistake of instancing the one on items though.

Reply:
fixed in new commit